recipe/meta.yaml

```diff
@@ -29,38 +29,41 @@ outputs:
         - docutils >=0.14,<1.0
       run:
         - python
-        - alembic >=0.8.3,<0.9
-        - bleach ==2.1.2
+        - alembic >=0.9,<1.0
         - configparser >=3.5.0,<3.6.0
         - croniter >=0.3.17,<0.4
         - dill >=0.2.2,<0.3
-        - flask >=0.12.4,<0.13
-        - flask-appbuilder >=1.11.1,<2.0.0
-        - flask-admin ==1.4.1
+        - enum34 ~=1.1.6  # [py<34]
+        - flask >=1.0,<2.0
+        - flask-appbuilder ==1.12.3
+        - flask-admin ==1.5.3
         - flask-caching >=1.3.3,<1.4.0
         - flask-login ==0.2.11
         - flask-swagger ==0.2.13
         - flask-wtf >=0.14.2,<0.15
         - funcsigs ==1.0.0  # [py<33]
         - future >=0.16.0,<0.17
         - gitpython >=2.0.2
-        - gunicorn >=19.4.0,<20.0
+        - gunicorn >=19.5.0,<20.0
         - iso8601 >=0.1.12
-        - jinja2 >=2.7.3,<2.9.0
+        - json-merge-patch ==0.2
+        - jinja2 >=2.7.3,<=2.10.0
         - lxml >=3.6.0,<4.0
         - markdown >=2.5.2,<3.0
         - pandas >=0.17.1,<1.0.0
         - pendulum ==1.4.4
-        - psutil >=4.2.0,<5.0.0
+        - psutil >=4.2.0,<6.0.0
         - pygments >=2.0.1,<3.0
         - python-daemon >=2.1.1,<2.2
         - python-dateutil >=2.3,<3
         - python-nvd3 ==0.15.0
-        - requests >=2.5.1,<3
+        - requests >=2.20.0,<3
         - setproctitle >=1.1.8,<2
-        - sqlalchemy >=1.1.15,<1.2.0
-        - tabulate >=0.7.5,<0.8.0
-        - tenacity ==4.8.0
+        - sqlalchemy >=1.1.15,<1.3.0
+        - tabulate 0.7.5,<0.9
+        - tenacity ==4.12.0
+        - text-unidecode ==1.2
+        - typing  # [py<35]
```

Comment:
@sondre, this is following the original `setup.py`
https://github.com/apache/airflow/blob/1.10.3/setup.py#L325

If you think it's also needed for python 3.7, let me know.

Reply:
@xylar, I am okay not building for python 3.7. 

Feel free to merge.

Reply:
Okay, I'll go ahead and merge.  We could always do another build that supports python 3.7 once the other two libraries are in place.